cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,126 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
+
 describe('Settings page', () => {
+  let user;
+  let newUser;
+
   before(() => {
+    cy.task('generateUser')
+      .then((generatedUser1) => {
+        newUser = generatedUser1;
+      });
 
+    cy.task('generateUser')
+      .then((generatedUser2) => {
+        user = generatedUser2;
+      });
   });
 
   beforeEach(() => {
+    cy.task('db:clear');
 
+    cy.loginAndRegister(
+      user.email,
+      user.username,
+      user.password);
+
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage
+      .typeUserName(newUser.username);
+
+    settingsPage
+      .clickUpdateBtn();
 
+    cy.get('.swal-button')
```

Comment:
Here you can use page object too. Besides I see that this element is repeated further